docs/hardware/babble.mdx

```diff
@@ -0,0 +1,56 @@
+# Project Babble (Face Tracking)
+
+## Summary
+
+[Project Babble](https://github.com/SummerSigh/ProjectBabble) is an open-source software project aiming to bring accessible face-tracking to PCVR headsets.
+
+## Set-up
+
+Setting up Project Babble is fairly simple. Please read carefully to avoid any issues. 
+
+:::note
+This guide assumes you have Project Babble installed, be it a release or the source code.
+It also assumes you have a compatible camera device. A standard webcamera will work, but you can also use a phone, etc.
```

Comment:
`webcamera` -> `webcam`

Reply:
Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).